rust_dev_preview/s3/README.md

```diff
@@ -30,225 +38,42 @@ amount of data at any time, from anywhere on the web.*
 - [Lists your buckets and uploads a file to a bucket](src/bin/s3-helloworld.rs) (ListBuckets, PutObject)
 - [Lists your buckets at a specified endpoint](src/bin/s3-object-lambda.rs) (ListBuckets)
 - [Uses an SQL expression to retrieve content from an object in a bucket](src/bin/select-object-content.rs) (SelectObjectContent)
+- [Create multipart upload](src/bin/s3-multipart-upload.rs)(CreateMultipartUpload)
+- [Upload part](src/bin/s3-multipart-upload.rs)(UploadPart)
+- [Complete multipart upload](src/bin/s3-multipart-upload.rs)(CompleteMultipartUpload)
 
-## ⚠ Important
-
-- We recommend that you grant this code least privilege, 
-  or at most the minimum permissions required to perform the task.
-  For more information, see
-  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
-  in the AWS Identity and Access Management User Guide.
-- This code has not been tested in all AWS Regions.
-  Some AWS services are available only in specific
-  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
-- Running this code might result in charges to your AWS account.
-
-### Prerequisites
-
-You must have an AWS account, and have configured your default credentials and AWS Region as described in the [Getting Started](https://github.com/awslabs/aws-sdk-rust#getting-started-with-the-sdk) section of the SDK README.
-The minimum version of Rust needed to run the SDK is listed [here](https://github.com/awslabs/aws-sdk-rust#supported-rust-versions-msrv).
-Instructions for installing Rust and Cargo can be found in the [Official Rust Documentation](https://doc.rust-lang.org/book/ch01-01-installation.html).
-
-## Running the code examples
+### Scenarios
 
```

Comment:
Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.

Reply:
Fixed.